README.md

```diff
@@ -474,7 +472,7 @@ MIT
 [roadmap-badge]: https://img.shields.io/badge/%F0%9F%93%94-roadmap-CD9523.svg?style=flat-square
 [roadmap]: https://github.com/paypal/glamorous/blob/master/other/ROADMAP.md
 [examples-badge]: https://img.shields.io/badge/%F0%9F%92%A1-examples-8C8E93.svg?style=flat-square
-[examples]: https://github.com/paypal/glamorous/blob/master/other/EXAMPLES.md
+[examples]: https://github.com/paypal/glamorous/blob/master/examples/README.md
```

Comment:
On second thought, instead of linking to the README, let's link to the directory itself (and github will render the README, but people will also see the other folders).

So:

```markdown
[examples]: https://github.com/paypal/glamorous/blob/master/examples
```